src/client/styles/components/Search.scss

```diff
@@ -6,12 +6,129 @@
 
   .nypl-text-field {
     input[type=text] {
-      height: 2rem;
       -webkit-box-shadow: none;
       box-shadow: none;
     }
   }
 
   .nypl-field-error {
   }
+
+  // Larger screen collapsed version of search form elements.
+
+  .nypl-omnisearch {
+    background-color: #fff;
+    margin-left: 0.25rem;
+  }
+
+  .nypl-omnisearch {
+    .nypl-omni-fields {
```

Comment:
You can further reduce this to:

```
  .nypl-omnisearch {
    background-color: #fff;
    margin-left: 0.25rem;

    .nypl-omni-fields {
      display: block;
      border-right: 0.08333rem solid #080807;
      float: left;
      width: 28%;
    }
  }
```

But I imagine it was to not expand to more than three levels down?

Reply:
There was a place where I removed a more specific designation of input type since moving the changes up didn't affect the layout. But wherever I can consolidate, I'll make the change. Could always debate whether to apply changes to the most specific first and work up unless it's more than three...